docs/reference/esql/processing-commands/limit.asciidoc

```diff
@@ -2,12 +2,28 @@
 [[esql-limit]]
 === `LIMIT`
 
-The `LIMIT` processing command enables you to limit the number of rows:
+**Syntax**
 
 [source,esql]
 ----
-include::{esql-specs}/docs.csv-spec[tag=limit]
+LIMIT max_number_of_rows
 ----
 
-If not specified, `LIMIT` defaults to `500`. A single query will not return
-more than 10,000 rows, regardless of the `LIMIT` value.
+*Parameters*
+
+`max_number_of_rows`::
+The maximum number of rows to return.
+
+*Description*
+
+The `LIMIT` processing command enables you to limit the number of rows that are
+returned. If not specified, `LIMIT` defaults to `500`.
+
+A query does not return more than 10,000 rows, regardless of the `LIMIT` value.
```

Comment:
out of scope: It may be worth noting that there are settings that can be overridden to adjust these values; in fact, Nhat and I just made the settings dynamic, so one can update them with a `PUT /_cluster/settings`.

Reply:
Oh, that's good to know! Thanks. I'll add that to the Limitations docs.

Reply:
The settings are called `esql.query.result_truncation_max_size` and `esql.query.result_truncation_default_size`.

I'm currently trying to backport a PR that correctly enables setting them on 8.11 as well.